src/semeio/fmudesign/design_distributions.py

```diff
@@ -147,17 +147,34 @@ def _check_dist_params_logunif(dist_params):
     return status, msg
 
 
-def draw_values_normal(dist_parameters, numreals, rng, normalscoresamples=None):
-    """Draws values from normal or truncated normal.
-    Args:
-        dist_parameters(list): [mean, std dev, min, max],
-        min/max defining truncated normal
-        numreals(int): number of realisations to draw
-        rng: numpy.random.RandomState instance
-        normalscoresamples(list): samples for correlated parameters
+def generate_stratified_samples(numreals, rng):
+    """Generate stratified samples in [0,1] by dividing the interval
+    into equal-probability strata.
+
+    This is equivalent to one-dimensional Latin Hypercube Sampling
+    where the [0,1] interval is divided into numreals equal segments,
+    and one random sample is drawn from each segment.
+
+    Parameters:
+        numreals: int, number of samples to generate
+        rng: numpy.random.RandomState, random number generator instance
+
     Returns:
-        list of values
+        numpy.ndarray: Array of stratified samples in [0,1]
     """
+    if numreals < 0:
+        raise ValueError("numreal must be a positive integer")
+
+    if numreals == 0:
+        return np.array([])
+
+    sampler = qmc.LatinHypercube(d=1, seed=rng)
+    samples = sampler.random(n=numreals)
+    scaled = qmc.scale(samples, 0, 1)
```

Comment:
not sure if this line does anything, since samples are already in the `[0, 1)` interval?